usagov/usagov-2021
# Usagov 1548 ceo state and country

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1548

## Description

Changed the name of the input and the autofill type so that users see the correct state when autofilling on Chrome. Check if the browser is Chrome (note this fires and fixes the problem in Edge as well) and only apply changes there. 
Firefox should still default to an empty dropdown and Safari does not autofill at all. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- make sure you have some addresses already saved to Chrome: (chrome://settings/addresses)
- on /elected-officials use the autofill tool from Chrome
- the correct state should show 
- click find my elected officials
- check that the address is correct 
- in Chrome address bar check that the query value for "select-dropdown" is also correct 
-  Check in other browsers (from what I can tell Safari can autofill the other fields but not state) 

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
